src/domains/wallet/pages/WalletDetails/components/WalletVote/WalletVote.tsx

```diff
@@ -54,12 +54,8 @@ export const WalletVote = ({ votes, maxVotes, isLoading, onButtonClick }: Wallet
 						<div className="flex flex-col mr-4 font-semibold leading-snug text-theme-text">
 							<span className="mr-2">{t("WALLETS.PAGE_WALLET_DETAILS.VOTES.EMPTY_DESCRIPTION")}</span>
 
-							<div className="mr-auto">
-								<Link
-									to="https://guides.ark.dev/usage-guides/desktop-wallet-voting"
-									isExternal
-									showExternalIcon={false}
-								>
+							<div className="mr-auto mt-1">
+								<Link to="https://guides.ark.dev/usage-guides/desktop-wallet-voting" isExternal>
```

Comment:
@goga-m change the link to this: https://ark.dev/docs/desktop-wallet/user-guides/how-to-vote-unvote

Reply:
Thanks @brenopolanski !